### ISSUE TITLE
Failing on windows 10 

### ISSUE THREAD

**Issue by kunle-e** [typicode/husky#76]
Node v4.24
npm v3.8.3
git  v1.9.5.msysgit.1

The generated logic in the post-commit hook looks like this:
[ -f package.json ] && cat package.json | grep '"postcommit"\s*:'
[ $? -ne 0 ] && exit 0

What's happening is that the \s*: is not working as expected. It will find nothing and exit 0.
If I change grep '"postcommit"\s*\:'    to     grep '"postcommit"[[:space:]]*\:'   It works.
Maybe this is more of a git bash issue, but I'm raising awareness.

**Comment by thm1118** [comment]
on windows , should use find command instead of  grep

**Comment by kentcdodds** [comment]
How can we help @typicode? I asked @gtramontina to deprecate `ghooks` so we could converge on a single githooks solution. I didn't realize that husky didn't support (non-bash) windows. What would you like to see by way of a solution? Any tips?

**Comment by typicode** [comment]
Thanks for the help and the report. I just gave a try now on Windows 10 and would need steps to reproduce.

With Git (git version 2.11.0.windows.1) installed using the installer from https://git-scm.com/ it works. The same goes for the GitHub client.

How should I install Git to have a failing case?

To test, I'm doing this:

```sh
npm init
git init
# edit package.json to add "precommit": "exit 1"
git add .
git commit -m # if there's some npm output, precommit hook worked
```

**Comment by typicode** [comment]
I've updated hook script to use `[[:space:]]`, this should broaden Windows support and make it work with older Git (`1.9.5.msysgit.1`).

@kunle-e thanks for the tip ;)

It's available in husky `v0.13.0`
